src/hypothesis/internal/escalation.py

```diff
@@ -74,9 +74,7 @@ def escalate_hypothesis_internal_error():
         raise
     import traceback
     filepath = traceback.extract_tb(tb)[-1][0]
-    if is_hypothesis_file(filepath) and not isinstance(
-        e, (HypothesisException,) + HYPOTHESIS_CONTROL_EXCEPTIONS,
-    ):
+    if is_hypothesis_file(filepath) and not isinstance(e, HypothesisException):
```

Comment:
Possibly I’ve missed something – why do we no longer exclude the control exceptions? StopTest doesn’t subclass from HypothesisException.

Reply:
Ugh, rebasing error :cry: